<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlacesQgisDialogBase</class>
 <widget class="QDialog" name="PlacesQgisDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Places Qgis</string>
  </property>
  <widget class="QLineEdit" name="gapiKey">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Google API key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>xlsx file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xlsxFilePath">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="xlsxFilePicker">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="latitude">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="longitude">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>latitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>longitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>keyword</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="logBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>471</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveLogCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Log?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radius">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>radius (kms)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeLayers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Layers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeWindows">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Windows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputDirName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="outputDirPicker">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveImages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>save images?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="limitEntries">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>limit entries</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
